python/ql/src/meta/ClassHierarchy/Find.ql

```diff
@@ -328,7 +328,7 @@ class FabricConnection extends FindSubclassesSpec {
   FabricConnection() { this = "fabric.connection.Connection~Subclass" }
 
   override API::Node getAlreadyModeledClass() {
-    result = FabricV2::Fabric::Connection::ConnectionClass::classRef()
+    result  = any(FabricV2::Fabric::Connection::ConnectionClass::Instance i)
```

Comment:
This is not right. `classRef` should be a reference to the _class_, whereas `Instance` is for _instances_ of said class. These are not the same thing, and this is important for this file, as it is explicitly about finding references to (subclasses of) known classes, _not_ instances of those classes.

Please just put `classRef()` back in place in `Fabric.qll`.

(Also, don't forget to autoformat your code. This file and `Torch.qll` failed the autoformat check.)